src/main/java/com/microsoft/lst_bench/exec/SessionExec.java

```diff
@@ -26,4 +26,7 @@ public interface SessionExec {
   String getId();
 
   List<TaskExec> getTasks();
+
+  /** Connection manager for this session (positional index). */
+  int getConnectionManager();
```

Comment:
What do you think about using string based identifiers for cluster connections for readability (cluster-1 is read-cluster and cluster-2 is maintenance) and consistency (for instance, workload operations like session, and tasks are also identified using strings)

Reply:
I had thought about this but I'm not convinced since it will make reusability of workloads harder, i.e., now we need to use the same identifier in workload to match the provided connection config. I think optimal path is having both options, I will create a GH issue for future work and link it here.

Reply:
See #79.